ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -311,7 +312,7 @@ object BigDiffy extends Command with Serializable {
   /** Merge two BigQuery TableSchemas. */
   def mergeTableSchema(x: TableSchema, y: TableSchema): TableSchema = {
     val r = new TableSchema
-    r.setFields(mergeFields(x.getFields.asScala, y.getFields.asScala).asJava)
+    r.setFields(mergeFields(x.getFields.asScala.toList, y.getFields.asScala.toList).asJava)
```

Comment:
Why this change? Iterator laziness?

Reply:
This expects a `Seq` which in 2.13 is an `scala.collection.immutable.Seq`.  `asScala` returns a `mutable.Buffer[TableFieldSchema]` which extends `mutable.Seq`.  It worked in 2.12 because the default Seq was `scala.collection.Seq` which they both inherit from.  Using toList instead of toSeq so it isn't seen as a redundant method call in 2.12